tests/maskers/test_tabular.py

```diff
@@ -54,7 +54,7 @@ def test_serialization_independent_masker_numpy():
         temp_serialization_file.seek(0)
 
         # deserialize masker
-        new_independent_masker = shap.maskers.Masker.load(temp_serialization_file)
+        new_independent_masker = shap.maskers.Independent.load(temp_serialization_file)
```

Comment:
Maybe I misinterpreted what this test is doing. I assumed that we want to use the load method specifically of the masker being tested, rather than the parent `Masker` class. Happy to revert it if needed. This and line 110 below are changed:
https://github.com/shap/shap/blob/43d47f63e9f747243f2f3f56fadda87265e75fe0/tests/maskers/test_tabular.py#L110